sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue

```diff
@@ -4,21 +4,26 @@
       <header-navigator
         :title="locationName"
         subTitle="Confirmation Of Enrolment"
+        data-cy="confirmationOfEnrolment"
```

Comment:
Are those not too generic (data-cy="confirmationOfEnrolment")?
Is it not better to have something more specific like `data-cy="confirmationOfEnrolmentHeader"`?
Maybe I am not following the idea. Can you please elaborate on that?

Reply:
I thought of keeping it more specific, but wanted to keep it consistent with the value in the `API` value in the formIO

Reply:
updated it